.github/workflows/daily-nightly-jobs.yml

```diff
@@ -394,64 +394,42 @@ jobs:
           name: ceph-upgrade-suite-quincy-artifact
           path: /home/runner/work/rook/rook/tests/integration/_output/tests/
 
-  rgw-multisite-test-with-ceph-devel:
```

Comment:
This test is part of the canary suite that is now getting kicked off (hopefully) so I think we can remove it.